src/components/notification/notification.module.scss

```
@@ -0,0 +1,147 @@
+    transform: translateY(0);
+  }
+}
+
+.icon {
+  display: flex;
+  flex-shrink: 0;
+}
+
+.message {
+  flex: 1;
+  font-size: pxToRem(14);
```

Stylelint (17.6.0)

[error] Expected "pxToRem" to be "pxtorem" (function-name-case)

(function-name-case)